OasisDEX/oasis-borrow
# Add TBTC Ajna pools

# [Add TBTC Ajna pools](https://app.shortcut.com/oazo-apps/story/10761/add-tbtc-pools)

![image](https://github.com/OasisDEX/oasis-borrow/assets/16230404/f1e07f71-8fea-4b34-9870-695751b9f113)
  
## Changes 👷‍♀️

- Added support for TBTC coin in the app,
- added suport TBTC/WBTC and TBTC/USDC Ajna pools,
- replaced WBTC in Product Finder with BTC group.
  
## How to test 🧪

- Run Product Finder PATCH,
- look for TBTC products,
- check open UI.

Also, ignore the duplication, I had to add `primaryTokenGroup: 'BTC',` in a lot of places.
